client.go

```diff
@@ -66,3 +66,98 @@ func (c *Client) UpdateFeatureState(featureState *FeatureState) (*FeatureState,
 	}
 	return &updatedFeatureState, nil
 }
+
+func (c *Client) GetProject(projectUUID string) (*Project, error) {
+	url := fmt.Sprintf("%s/projects/", c.baseURL)
+	result := []*Project{}
+	resp, err := c.client.R().
+		SetQueryParams(map[string]string{
+			"uuid": projectUUID,
+		}).
+		SetResult(&result).
+		Get(url)
+
+	if err != nil {
+		return nil, err
+	}
+	if !resp.IsSuccess() || len(result) != 1 {
+		return nil, fmt.Errorf("flagsmithapi: Error getting project: %s", resp)
+	}
+	project := result[0]
+	return project, nil
+
+}
+
+func (c *Client) GetFeature(featureUUID string) (*Feature, error) {
+	url := fmt.Sprintf("%s/features/get-by-uuid/%s/", c.baseURL, featureUUID)
+	feature := Feature{}
+	resp, err := c.client.R().
+		SetQueryParams(map[string]string{
+			"uuid": featureUUID,
+		}).
+		SetResult(&feature).Get(url)
+
+	if err != nil {
+		return nil, err
+	}
+
+	if !resp.IsSuccess() {
+		return nil, fmt.Errorf("flagsmithapi: Error getting feature: %s", resp)
+	}
+	return &feature, nil
+}
+
+func (c *Client) CreateFeature(feature *Feature) error {
+	projectID := feature.ProjectID
+	if projectID == nil {
+		project, err := c.GetProject(feature.ProjectUUID)
+		if err != nil {
+			return err
+		}
+		projectID = &project.ID
+	}
+
+	url := fmt.Sprintf("%s/projects/%d/features/", c.baseURL, *projectID)
+
+	resp, err := c.client.R().SetBody(feature).SetResult(&feature).Post(url)
+
+	if err != nil {
+		return err
+	}
+
+	if !resp.IsSuccess() {
+		return fmt.Errorf("flagsmithapi: Error creating feature: %s", resp)
+	}
+
+	return nil
+}
+
+func (c *Client) DeleteFeature(projectID, featureID int64) error {
```

Comment:
General Golang question... how come you have to specify that the featureID is `int64` but not the projectID?